src/collapse/docs/readme.md

```diff
@@ -7,3 +7,27 @@
   <i class="glyphicon glyphicon-eye-open"></i>
   _(Default: `false`)_ -
   Whether the element should be collapsed or not.
+
```

Comment:
I went off the uib-alert documentation which uses:
```
* `close`
  <small class="badge">$</small>
  _(Default: `none`)_ -
  A callback function that gets fired when an `alert` is closed. If the attribute exists, a close button is displayed as well.
```
But I'm happy to go with the format used by the uib-tab directive, if you prefer.